### PR TITLE
Swap Colorado in for Utah

### PR DESCRIPTION
**Pull request by zsusswein**
To avoid any any particular anomalies. This involved:

- [x] swapping `'co'` for `'ut'` in the rmd and the text
- [x] downloading fresh the data from covidcast and saving the new rds
- [x] rebuilding the report

It looks good on my end, but please check if I missed anything! If there's something
outside the rmarkdown, I definitely didn't change it.

### CODE REVIEW

**Comment by zsusswein** [comment]
@seabbs, this was at @kgostic's request, given the long Utah lag. Why don't you glance at this one and once it has your approval to merge, I'll also merge it into #12 and rebuild the report there.

File: renv.lock
Comment (seabbs) [inline renv.lock:38]: Other than this one these are all just because your not on Linux. 

This one is more of a problem because we want to install EpiNow2 from it's universe Vs from GitHub because then windows and Mac users get precompiled binaries. 

If you could revert this one that would be great.

Comment (seabbs) [reply renv.lock:38]: (I think you've introduced but not entirely clear to me.)

File: epinow2.Rmd
Comment (seabbs) [inline epinow2.Rmd:97]: Why drop entirely?

**Review by seabbs — COMMENTED** [review]
Just looking from my phone so might not be the best ever review!

File: epinow2.Rmd
Comment (zsusswein) [reply epinow2.Rmd:97]: It's the same sentence as in the states of interest section on para above. Seemed like it was fully address there?

File: renv.lock
Comment (zsusswein) [reply renv.lock]: Ah this was an accident -- I was just trying to add bookdown so that the report could be knit. Fixed.

File: epinow2.Rmd
Comment (seabbs) [reply epinow2.Rmd:97]: Ah okay.


Fair enough.

**Comment by seabbs** [comment]
Looks good to me. 

Agree bookdown needs to be in the lock file. I might burn it all down and go again now we are getting somewhere.

**Review by seabbs — APPROVED** [review]
LGTM

**Comment by zsusswein** [comment]
> Looks good to me.
> 
> Agree bookdown needs to be in the lock file. I might burn it all down and go again now we are getting somewhere.

I'm a strong vote yes on this -- probably better that it comes from you sometime tomorrow because my WSL2 is being wonky on the dependencies

**Comment by seabbs** [comment]
I'll do that and setup building the html on the CI so we know it works. 

@kgostic there is a choice here to either 

1. Leave the html where it is

2. Move it to a branch

3. Host it as an object

With 3. Being the most modern approach for GitHub pages. 

In terms of people being able to find stuff it goes from 1 to 3

In terms of repo size it goes from 3 to 1

Hangs on if people will look at the html via the web or by clicking on it from a zip. 

What does everyone think? 

Tomorrow I am going to purge the old html files and images from the history to make sure the repo gets a bit smaller but 2 and 3 would make this size reduction permanent. 

Personally I would go with 3. but only just vs keeping things simple with 1.

**Comment by kgostic** [comment]
@seabbs - I think #3 would work well.

I do not think most of the meeting participants know what GitHub is, so I am not really expecting them to be perusing the repo trying to find the .html file.

Zack and I haven't been able to get the binder deployment to work, so we are planning to email a .zip file of the whole repo out to the participants with instructions in how to open the .Rmd file. (I know this is going to kill you).

If #3 allows us to host the .html on GitHub pages and I can open it in my web browser during the demo, that would be excellent.

**Comment by kgostic** [comment]
@zsusswein this PR looks good. No changes needed for merge.

**Comment by seabbs** [comment]
It's already hosted on GitHub pages but yes 3 is a better way of doing it. 

It would be useful to know what "not been able to get it to work" means. 

It was only ever intended as a backup as the machines are hence and so slow.

**Comment by seabbs** [comment]
Rebased and looking good.

**Pull request merged**
